pkg/internal/apis/certmanager/validation/clusterissuer.go

```diff
@@ -30,3 +30,9 @@ func ValidateClusterIssuer(obj runtime.Object) field.ErrorList {
 	allErrs := ValidateIssuerSpec(&iss.Spec, field.NewPath("spec"))
 	return allErrs
 }
+
+func ValidateUpdateClusterIssuer(oldObj, obj runtime.Object) field.ErrorList {
+	iss := obj.(*cmapi.ClusterIssuer)
+	allErrs := ValidateIssuerSpec(&iss.Spec, field.NewPath("spec"))
+	return allErrs
+}
```

Comment:
Why do we need to introduce update validation in this PR?
And why do we *not* do the CSR / CR usages validation in this case?
Please add an explanatory comment.